frontend/js/src/explore/art-creator/components/IconTray.tsx

```diff
@@ -83,6 +86,29 @@ function IconTray(props: IconTrayProps) {
             </span>
           )}
         </div>
+        {browserHasClipboardAPI && (
+          <button
+            type="button"
+            className="btn btn-icon btn-info"
+            onClick={onClickCopyAlt}
+            style={{
+              marginLeft: "auto",
+              background: "transparent",
+              color: "darkgray",
```

Comment:
In the same spirit as above if you don't want to create a new class for this button, these two style can be replaced by using these css classes for the button component : `btn btn-sm btn-link`

and then adding the class `text-muted` to both children elements (the text and the icon)